permissions/plugin-proxmox.yml

```diff
@@ -4,4 +4,5 @@ github: "jenkinsci/proxmox-plugin"
 paths:
 - "org/jenkins-ci/plugins/proxmox"
 developers:
-- "justnom"
\ No newline at end of file
+- "justnom"
+- "maxhy"
```

Comment:
`User name not known to Artifactory: maxhy`

this needs to be your Jenkins infra username (e.g., Jira login)

Reply:
Mhhh that's my username and I previously logged in on Artifactory. Maybe it takes a few hours to sync? I repushed the commit without the new line.